app/views/static_pages/mtbachelor.html.erb

```diff
@@ -0,0 +1,202 @@
+  <!-- HTML5 Shim and Respond.js IE8 support of HTML5 elements and media queries -->
+  <!-- WARNING: Respond.js doesn't work if you view the page via file:// -->
+  <!--[if lt IE 9]>
+  <script src="https://oss.maxcdn.com/libs/html5shiv/3.7.0/html5shiv.js"></script>
+  <script src="https://oss.maxcdn.com/libs/respond.js/1.4.2/respond.min.js"></script>
+  <![endif]-->
+</head>
+<body>
+
+<!-- Page Content -->
+<div class="container">
+  <!-- Page Heading/Breadcrumbs -->
+  <div class="row">
+    <div class="col-lg-12">
+      <h1 class="page-header">
+        Mt. Bachelor
+      </h1>
+    </div>
+  </div>
+  <!-- /.row -->
+  <!-- Portfolio Item Row -->
+  <div class="row">
+    <div class="col-md-8">
+      <div id="carousel-example-generic" class="carousel slide" data-ride="carousel">
+        <!-- Indicators -->
+        <ol class="carousel-indicators">
+          <li data-target="#carousel-example-generic" data-slide-to="0" class="active"></li>
+          <li data-target="#carousel-example-generic" data-slide-to="1"></li>
+          <li data-target="#carousel-example-generic" data-slide-to="2"></li>
+        </ol>
+        <!-- Wrapper for slides -->
+        <div class="carousel-inner">
+          <div class="item active">
+            <img class="img-responsive" src="http://radventurepdx.github.io/img/mtbachelor.jpg" alt="">
+          </div>
+          <div class="item">
+            <img class="img-responsive" src="http://radventurepdx.github.io/img/bachelor2.jpg" alt="">
+          </div>
+          <div class="item">
+            <img class="img-responsive" src="http://radventurepdx.github.io/img/bachelor3.jpg" alt="">
+          </div>
+        </div>
+        <!-- Controls -->
+        <a class="left carousel-control" href="#carousel-example-generic" data-slide="prev">
+          <span class="glyphicon glyphicon-chevron-left"></span>
+        </a>
+        <a class="right carousel-control" href="#carousel-example-generic" data-slide="next">
+          <span class="glyphicon glyphicon-chevron-right"></span>
+        </a>
+      </div>
+    </div>
+    <div class="col-md-4">
+      <h3>Location Description</h3>
+      <p>Lorem ipsum dolor sit amet, consectetur adipiscing elit. Nam viverra euismod odio, gravida pellentesque urna varius vitae. Sed dui lorem, adipiscing in adipiscing et, interdum nec metus. Mauris ultricies, justo eu convallis placerat, felis enim.</p>
+      <h3>Location Details</h3>
+      <ul>
+        <li>Lorem Ipsum</li>
+        <li>Dolor Sit Amet</li>
+        <li>Consectetur</li>
+        <li>Adipiscing Elit</li>
+      </ul>
+    </div>
+  </div>
+  <!-- /.row -->
+  <!-- Related Projects Row -->
+  <div class="row">
+    <div class="col-lg-12">
+      <h3 class="page-header">Top Guides for Mt. Bachelor</h3>
+    </div>
+
+    <% top_guides = [] %>
+    <% @guides = Guide.all %>
+    <% @guides.where(:location => 'Mount Bachelor').each do |guide| %>
```

Comment:
I would move these three lines to a method in guides_helper file since this is a logic and it shouldn't really be in view pat.